src/licensedcode/license_db.py

```diff
@@ -200,6 +200,14 @@ def scancode_license_data(path):
     Dumps data in JSON, YAML and HTML formats and also dumps the .LICENSE file
     with the license text and the data as YAML frontmatter.
     """
+    if not path:
```

Comment:
Thanks... but a simpler approach would be with https://github.com/aboutcode-org/scancode-toolkit/issues/4024#issuecomment-2551065436